include/z64player.h

```diff
@@ -7,38 +7,38 @@
 struct Player;
 
 typedef enum {
-    /* 0x00 */ PLAYER_SHIELD_NONE,
-    /* 0x01 */ PLAYER_SHIELD_HEROS_SHIELD,
-    /* 0x02 */ PLAYER_SHIELD_MIRROR_SHIELD,
-    /* 0x03 */ PLAYER_SHIELD_MAX
+    /* 0x0 */ PLAYER_SHIELD_NONE,
+    /* 0x1 */ PLAYER_SHIELD_HEROS_SHIELD,
+    /* 0x2 */ PLAYER_SHIELD_MIRROR_SHIELD,
+    /* 0x3 */ PLAYER_SHIELD_MAX
 } PlayerShield;
```

Comment:
```suggestion
    /* 0 */ PLAYER_SHIELD_NONE,
    /* 1 */ PLAYER_SHIELD_HEROS_SHIELD,
    /* 2 */ PLAYER_SHIELD_MIRROR_SHIELD,
    /* 3 */ PLAYER_SHIELD_MAX
} PlayerShield;
```
There's a lot of single digit enums in `z64player.h`, I won't comment on them all